stable/keel/templates/clusterrole.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.rbac.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "keel.fullname" . }}
```

Comment:
I'd suggest you use `include` consistently instead of `template`. Right now it is somewhat random.

Reply:
I did that everywhere, even if the NOTES.txt but i'm not sure about this one